release/release_after_kokoro.sh

```diff
@@ -89,7 +89,8 @@ ask_proceed
 ###############################################################################
 echo
 echo "#"
-echo "# Run Eclipse from the command line to generate the following files:"
+echo "# Run Eclipse from the command line to generate a p2 repository from"
+echo "# the signed artifacts:"
```

Comment:
How about "the following p2 repository"? To prevent people from thinking that the signed artifacts are under `$LOCAL_REPO`.